sdk/v1/client.go

```diff
@@ -92,6 +92,31 @@ func (cli *Client) PostJob(ctx context.Context, headers client.Headers) (models.
 	return job, nil
 }
 
+// PostTasksCount Updates tasks count for processing
+func (cli *Client) PostTasksCount(ctx context.Context, headers client.Headers, jobID string, payload []byte) (models.Task, error) {
+	var task models.Task
+
+	if headers.ServiceAuthToken == "" {
+		headers.ServiceAuthToken = cli.serviceToken
+	}
+
+	path := fmt.Sprintf("%s/jobs/%s/tasks", cli.apiVersion, jobID)
```

Comment:
Something like this will be needed here: 

payload, _ := json.Marshal(taskToCreate)

Reply:
nope, see my comments as above